openwisp_monitoring/db/exceptions.py

```diff
@@ -0,0 +1,7 @@
+class TimeseriesWriteException(Exception):
+    """
+    Caused by excessive load on timeseries server
+    leading to failure in storing metrics
```

Comment:
this is misleading, we can't know the reason the tsdb is failing in advance, it may fail for any reason. The point is that when the system finds this exception will retry.

Reply:
I will remove it but I thought the reason for failure was known to us, as it was stated in the issue:~
> If for some reason the timeseries DB is overloaded, the write fails and the data is lost.

Reply:
you can never know the reason of all the possible failures that will happen when the service is used in the wild
the important bit is that these exceptions will be handled

Reply:
Agreed :smiley: